src/Cosmos/Serialization.cpp

```diff
@@ -23,7 +23,7 @@ const std::string AMINO_PREFIX_WITHDRAW_STAKE_MESSAGE = "cosmos-sdk/MsgWithdrawD
 const std::string AMINO_PREFIX_TRANSACTION = "auth/StdTx";
 const std::string AMINO_PREFIX_PUBLIC_KEY = "tendermint/PubKeySecp256k1";
 
-json broadcastJSON(json& jsonObj) {
+static json broadcastJSON(json& jsonObj) {
```

Comment:
Why change to static?

Reply:
because otherwise function with same name in other module causes linking error (duplicate symbol)

it's not strictly necessary for this PR to do this for Comsos code, but generally is a good practice for C++ functions internal to given module